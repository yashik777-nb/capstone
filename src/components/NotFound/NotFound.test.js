import React from "react";
import { configure, shallow, mount } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { createMount } from "@material-ui/core/test-utils";

import NotFound from "./NotFound";

configure({ adapter: new Adapter() });

describe("NotFound Tests", () => {
  let wrapper;

  beforeEach(() => {
    const render = createMount();
    wrapper = render(<NotFound />);
  });

  it("Should render one not found", () => {
    expect(wrapper.find("h1").length).toEqual(1);
  });

  test("Should Render Not Found page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
