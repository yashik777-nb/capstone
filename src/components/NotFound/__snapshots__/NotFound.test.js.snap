// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Tests Should Render Not Found page 1`] = `ReactWrapper {}`;
